server/index.js

```diff
@@ -134,33 +135,40 @@ server.put('/reviews/:reviewId', (req, res) => {
     });
 });
 
-server.delete('/reviews/:reviewId', (req, res) => {
-  pgdb.deleteReview(req.params.reviewId).then((result) => {
-    res.status(200).send(result);
-  });
+app.delete('/reviews/:reviewId', (req, res) => {
+  pgdb
+    .deleteReview(req.params.reviewId)
+    .then((result) => {
+      res.status(200).send(result);
+    })
+    .catch((err) => {
+      res.status(500).send(err);
+    });
 });
 
-server.get('/product', (req, res) => {
+app.get('/product', (req, res) => {
   const { itemID } = req.query;
```

Comment:
it's funny how even this change makes me feel much better in my soul.

Reply:
I know!!!